automation/taskcluster/lib/tasks.py

```diff
@@ -225,6 +329,31 @@ def _craft_build_ish_task(
             payload
         )
 
+    def _craft_dummy_task(
+        self, name, description,  dependencies=None, routes=[], scopes=[],
```

Comment:
Nit: Do not use mutable default arguments https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments, use `None` instead.

Reply:
Good catch!